includes/functions/content-sync.php

```diff
@@ -269,7 +269,7 @@ function get_post_data( $post ) {
 		'publishedAt'         => gmdate( \DateTime::RFC3339, strtotime( $post->post_date_gmt ) ),
 		'plainText'           => wp_strip_all_tags( $content ),
 		'size'                => str_word_count( wp_strip_all_tags( $content ) ),
-		'sectionNames'        => Utils\get_post_categories( $post->ID ),
```

Comment:
we want to keep this field as the next step in a subsequent release will change the data in the `sectionNames` field to hold the Primary Category